src/components/Select/Select.tsx

```diff
@@ -44,7 +47,7 @@ export const Select:React.FC<Props> = (props) => {
   const [isOpen, setIsOpen] = useState(false);
   const [selectedValue, setSelectedValue] = useState(selectedOption?.title || '');
   const rootRef = useRef<HTMLDivElement>(null);
-  const placeHolder = 'Select value';
+  const placeHolder = title === 'Sort by' ? SORTED_BY_DEFAULT : ITEMS_PER_PAGE_DEFAULT;
```

Comment:
placeHolder should be something like `select value`, and you should show placeholder if there is no selected option. If you have selected option you show that option.